### PR TITLE
voxpupuli-rubocop: Update to 2.5.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -238,6 +241,20 @@ RSpec/ReceiveNever:
   Exclude:
     - 'spec/beaker/hypervisor/vagrant_spec.rb'
 
+# Offense count: 7
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, IgnoreMetadata.
+# Include: **/*_spec.rb
+RSpec/SpecFilePathFormat:
+  Exclude:
+    - '**/spec/routing/**/*'
```
Comment (ekohl) [inline .rubocop_todo.yml:249]: Where does this come from?

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -307,6 +324,14 @@ Style/EnvHome:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: AllowedReceivers.
+# AllowedReceivers: Thread.current
+Style/HashEachMethods:
+  Exclude:
+    - 'lib/beaker/hypervisor/vagrant.rb'
```
Comment (ekohl) [inline .rubocop_todo.yml:333]: https://github.com/voxpupuli/beaker-vagrant/pull/82 should fix this

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -238,6 +241,20 @@ RSpec/ReceiveNever:
...
+RSpec/SpecFilePathFormat:
+  Exclude:
+    - '**/spec/routing/**/*'
+    - 'spec/beaker/hypervisor/vagrant_custom_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_desktop_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_fusion_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_libvirt_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_parallels_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_virtualbox_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_workstation_spec.rb'
```
Comment (ekohl) [inline .rubocop_todo.yml:256]: Should we fix this by renaming `spec/beaker/hypervisor/*` to `spec/beaker/*`?

File: .rubocop_todo.yml (hunk 1 above)
Comment (bastelfreak) [reply .rubocop_todo.yml:249]: so this is odd. the directory doesn't exist on my local filesystem. But if I remove it from the todo file and regenerate it, rubocop adds it back :thinking:

File: .rubocop_todo.yml (hunk 3 above)
Comment (bastelfreak) [reply .rubocop_todo.yml]: moved it

File: lib/beaker/hypervisor/vagrant.rb [hunk 4]
```diff
@@ -91,7 +91,7 @@ def make_vfile(hosts, options = {})
           end
         end
 
-        if /windows/i.match(host['platform'])
+        if /windows/i.match?((host['platform']))
```
Comment (ekohl) [inline lib/beaker/hypervisor/vagrant.rb:94]: These braces look redundant. Also repeated a few times

Comment (bastelfreak) [reply lib/beaker/hypervisor/vagrant.rb]: fixed it, looks good now.

**Review by sebastianrakel — APPROVED** [review]

**Pull request merged**